Scoop/Controller/ProfileViewController.swift

```diff
@@ -189,7 +193,7 @@ class ProfileViewController: UIViewController {
         }
         
         let quietLabel = UILabel()
-        quietLabel.font = .systemFont(ofSize: 12)
+        quietLabel.font = .systemFont(ofSize: 14)
```

Comment:
Why is `quietLabel` a different font than `travelingLabel`? This goes for all labels...are there supposed to be different font types?

Reply:
Yep, quiet label is the label on the sliders so its smaller, but the traveling label is the header for the section